doc/classes/AudioStream.xml

```diff
@@ -39,6 +39,12 @@
 				Returns the length of the audio stream in seconds.
 			</description>
 		</method>
+		<method name="instance_playback">
+			<return type="AudioStreamPlayback" />
+			<description>
+    				Returns an [AudioStreamPlayback]. Useful when you want to extend [method _instance_playback] but call [code]instance_playback[/code] from another [AudioStream].
```

Comment:
It's not really obvious what this means from the current description. But I guess the virtual methods need to be documented too for this to make sense.

Reply:
There's a style issue (too many tabs for indent).